packages/asset-service/src/service/descriptions/es.json

```diff
@@ -1,7 +1,8 @@
-  "eip155:1/erc20:0x55a8a39bc9694714e2874c1ce77aa1e599461e18": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el grupo y también pueden depositar su token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd). TerraUSD ([UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd)) es la moneda estable algorítmica y descentralizada de la cadena de bloques Terra. Para adquirir Curve MIM-UST, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd) en [Curve](https://curve.fi/factory/48/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanente, pero debido a que todos los activos en el fondo están vinculados a $1, la pérdida impermanente es poco probable y solo se produciría si uno de los activos en el pool perdió su anclaje.",
+  "eip155:1/erc20:0x55a8a39bc9694714e2874c1ce77aa1e599461e18": "Este token representa un grupo de liquidez en Curve, un exchange descentralizado líder. Los titulares obtienen tarifas de los usuarios que comercian en el grupo y también pueden depositar su token LP en [ShapeShift](/defi/earn) para obtener rendimiento. Este grupo de liquidez contiene [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd). TerraUSD([UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd)) es la moneda estable algorítmica y descentralizada de la cadena de bloques Terra. Para adquirir Curve MIM-UST, deposite [MIM](/assets/eip155:1/erc20:0x99d8a9c45b2eca8864373a26d1459e3dff1e17f3) y [UST](/assets/eip155:1/erc20:0xa47c8bf37f92abed4a126bda807a7b7498661acd) en [Curve](https://curve.fi/factory/48/deposit). Tenga en cuenta que, dado que los fondos de liquidez están compuestos por activos de diferentes precios, están sujetos a pérdidas impermanente, pero debido a que todos los activos en el fondo están vinculados a $1, la pérdida impermanente es poco probable y solo se produciría si uno de los activos en el pool perdió su anclaje.",
```

Comment:
UST is now USTC, pls update references accordingly 
<img width="653" alt="image" src="https://user-images.githubusercontent.com/17035424/181820800-e0b02f93-3165-4cea-984f-b8d7e7c66223.png">

Reply:
Updating... en.json must be updated too